### PR TITLE
Throttle resize event and fix sidebar disappearance

### LINKED ISSUE #736: [Mobile] Sidebar disappears  when clicked in search box
This is happening in Chrome browser for Android, in hexdocs.pm

### PR DESCRIPTION
**Pull request by nscyclone**
Closes #736.

### CODE REVIEW

File: assets/js/helpers.js [hunk 1]
```diff
@@ -33,3 +33,16 @@ export function findSidebarCategory (items, query) {
 export function getLocationHash () {
   return window.location.hash.replace(/^#/, '')
 }
+
+export function throttle (callback, limit) {
```
Comment (dignifiedquire) [inline assets/js/helpers.js]: you could use `lodash.throttle` instead https://lodash.com/docs/4.17.4#throttle

Comment (nscyclone) [inline assets/js/helpers.js]: Thanks, @dignifiedquire!
I thought about it but for some reason didn't check whether we include the whole `lodash` or just some of its parts and decided to reinvent a wheel. I've updated the PR now.

**Comment by nscyclone** [comment]
Sorry for bothering, are there any more changes needed for this PR?
Or maybe the proposed changes are unwanted and I should close it?
/cc @dignifiedquire @josevalim

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by josevalim** [comment]
Sorry for dropping the ball on this one and thanks for the ping! :heart:

**Pull request merged**